enclave_build/src/storage.rs

```diff
@@ -567,4 +595,29 @@ pub mod tests {
 
         assert_eq!(config, expected_config);
     }
+
+    #[test]
+    fn test_get_image_hash_from_name() {
+        let (_root_path, mut storage) = setup_temp_storage();
+
+        let image_details = storage
+            .fetch_image_details(image::tests::TEST_IMAGE_NAME)
```

Comment:
Is self.config here None or Some? Can we test here the other scenario as well, e.g. if self.config is Some, the config is preserved?

Reply:
I'm pretty sure that the config is `Some` after performing `setup_temp_storage` (does the store as well). Do we want to add a test with `None`?

Reply:
Ideally every code path should be tested, so if it's not too difficult, yes we should add a test for that.